components/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Image from "next/image";
+import logo from "../public/images/Logo.svg";
+import LoginBtn from "./utility-components/LoginBtn";
+function Navbar() {
+  return (
+    <div className=" fixed right-0 left-0 z-20 bg-[white] flex py-2 px-10 justify-between shadow-xl">
+      <Image className=" h-10 w-10" src={logo} />
+
+      <div className="flex justify-evenly w-[50%] items-center">
```

Comment:
<img width="729" alt="image" src="https://user-images.githubusercontent.com/45912180/200631910-24ff2873-d28d-499d-b96b-d30675edc797.png">

This is too much space between the button, let the left flex-item (Onw with logo image) grow as much as possible and width of buttons in navbar will depend on the content inside just put gap of 1 or 2 rem